.circleci/config.yml

```
@@ -11,7 +11,7 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: pipenv-{{ checksum "Pipfile.lock" }}-v1
+          key: pipenv-{{ checksum "Pipfile.lock" }}-v2
```

```suggestion
          key: pipenv-{{ checksum "Pipfile.lock" }}-v1
```
After rebasing, the increment should no longer be required (but yeah, it doesn't hurt either :laughing:)

---

I know, but wanted to do the rebase close to merging so I dont have to do it again when things are merged before this one :smile:

---

Ah ok :smile: 
Probably you already knew, but just in case you didn't: I wrote a small [bash script](https://github.com/digitalfabrik/integreat-cms/blob/develop/dev-tools/resolve_translation_conflicts.sh) to resolve the conflicts in the translation file automatically, so as long as you only have conflicts in the translation file (and you only have one commit), you can rebase automatically by executing:
```
git fetch --all && GIT_SEQUENCE_EDITOR=true git rebase origin/develop || ./dev-tools/resolve_translation_conflicts.sh && GIT_EDITOR=true git rebase --continue && git push --force-with-lease
```